doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -31,6 +31,44 @@ There are multiple notions of :gdef:`equality` in Coq:
   other advantages.  ":term:`Convertible <convertible>`" is another way of saying that
   two terms are definitionally equal.
 
+Equality
+--------
+
+.. tacn:: f_equal
+
+   For a goal with the form :n:`f a__1 ... a__n = g b__1 ... b__n`, creates
+   subgoals :n:`f = g` and :n:`a__i = b__i` for the `n` arguments. Subgoals
+   that can be proven by :tacn:`reflexivity` or :tacn:`congruence` are solved
+   automatically.
```

Comment:
My feeling is that this tactic is more advanced than `reflexivity`, `symmetry` and `transitivity` and should therefore appear after them.